scopes/component/ui/component-meta/component-overview.module.scss

```diff
@@ -39,3 +39,8 @@
 .copyBox {
```

Comment:
replace .nav class with this:
```
.nav > div {
    font-size: 12px;
    padding: 6px 12px;
  }
```

Reply:
the block need to look like this:
![image](https://user-images.githubusercontent.com/35892475/204132935-f06d8e67-fbd5-4eb2-aacc-2a2c988ff9ef.png)
and now it's look like this:
![image](https://user-images.githubusercontent.com/35892475/204132957-041f8a82-e651-446b-ae2c-73f695831784.png)
with double border and bad border radius and wrong font-size and padding.